src/views/listview/examples/list-view.js

```diff
@@ -318,6 +320,30 @@
           }
         };
 
+        $scope.emptyStateActionButtons = [
+          {
+            name: 'Main Action',
+            title: 'Perform an action',
+            actionFn: performAction,
```

Comment:
Hi, I noticed in these examples the performAction doesn't update the Events text area.  I think because the callback function for emptyState action buttons only return an 'action' and not a 'item', so you can't reuse the existing performatAction(action, item) function already there.